### ISSUE TITLE
[HW] Kumaraswami Muthuswami

### ISSUE THREAD

**Issue by kumaraswami** [datastaxdevs/workshop-spring-stargate#2]
**Name:** Kumaraswami Muthuswami
**Email:** swami.mig@gmail.com
**Linkedin Profile:** https://www.linkedin.com/in/kumaraswami-muthuswami-0b73627/

Attach the homework step I screenshots below:
-----------------------------------------

![image](https://user-images.githubusercontent.com/20376524/116453302-54cde200-a824-11eb-98af-1d77885cc121.png)
  
Attach the homework step II screenshots below:
-----------------------------------------

![image](https://user-images.githubusercontent.com/20376524/116453302-54cde200-a824-11eb-98af-1d77885cc121.png)

**Comment by kumaraswami** [comment]
Thanks for the wonderful session

**Comment by SonicDMG** [comment]
Thank you so much for saying so.

**Comment by SonicDMG** [comment]
Were you able to get through the app as well. Do you have those screenshots? If so, please provide them here. Thank you.

**Comment by SonicDMG** [comment]
Yes I am

**Comment by kumaraswami** [comment]
Yes I was able to get through the app following the instructions the git, but I forgot to take and attache the screen shot of the stargate-demo app in the homework.

**Comment by SonicDMG** [comment]
Ok, no worries. I'll trust you did the work. I would have helped if we check your submission 2 weeks ago anyway. :)

**Comment by HadesArchitect** [comment]
Congrats! Well done! https://api.badgr.io/public/assertions/krfr8u00Q7atOxLVWin5Dw?identity__email=swami.mig%40gmail.com
